stock_mts_mto_mrp/wizards/change_production_qty.py

```diff
@@ -10,6 +10,7 @@ class ChangeProductionQty(models.TransientModel):
     @api.multi
     def change_prod_qty(self):
         origs = {}
+        move_orig = self.mo_id.move_raw_ids[0].move_orig_ids
```

Comment:
I don't know if this is correct, what will happen if production has more than 1 component??